data/json/items/armor/eyewear.json

```
@@ -205,6 +205,24 @@
     "flags": [ "WATER_FRIENDLY", "OUTER" ],
     "armor": [ { "encumbrance": 5, "coverage": 95, "covers": [ "eyes" ] } ]
   },
+  {
+    "id": "glass_goggles",
+    "type": "ARMOR",
+    "name": { "str": "pair of glass goggles", "str_pl": "pairs of glass goggles" },
+    "description": "A pair of old-fashioned goggles with thick glass lenses, somewhat similar to those worn by the pilots of open-cockpit aircraft, but more crude looking.  Wouldn't protect your eyes from physical harm, but they could keep hazardous chemicals out.",
```

wouldn't doesn't really make sense
```suggestion
    "description": "A pair of old-fashioned goggles with thick glass lenses, somewhat similar to those worn by the pilots of open-cockpit aircraft, but more crude looking.  Won't protect your eyes from physical harm, but can keep hazardous chemicals out.",
```